Game/Sets/73f1d13a-38cf-46b9-a661-a7f1a1413363/set.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Note to self: Format those set, so it's easier to see what actually changed...
So what did change here, though? Chances are high that the changes will be gone once I'll make a new version. Currently the sets are auto-generated from a script using information from a database, so they are always wiped prior to release. I'll probably have to include all those scripts at some point.

Reply:
I don't think anything changed here. I am using Notepad++ so it's possible some whitespace issue? I don't know honestly.